python/shotgun_desktop/wss_back_compat.py

```diff
@@ -0,0 +1,351 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import struct
+import os
+import sys
+import traceback
+
+from .desktop_message_box import DesktopMessageBox
+
+logger = None
+
+
+class Settings(object):
```

Comment:
This file is basically a dump of all the old websocket init code we had. It's kept in a single file so its easy to find and kill when we deprecate the legacy websocket integration entirely.